src/govuk/components/input/_index.scss

```diff
@@ -39,6 +39,13 @@
         border-width: $govuk-border-width-form-element * 2;
       }
     }
+
+    &:disabled {
+      opacity: .5;
+      color: inherit;
+      background-color: transparent;
```

Comment:
What does the background colour do here out of interest?

With us not having one on `.govuk-input`

Reply:
Chrome and Internet Explorer—and only those two browsers, for whatever reasons—apply a light grey background-color to disabled text inputs and textareas. Here I've opted to remove it so that the background-color is consistent across all text inputs on all browsers.

Reply:
Ahh that's good to know

In a world of "extended components" in the various organisational-level design systems, I'm wondering if this will make some department's (blue|green|red|whatever)-tinted input suddenly go transparent when disabled?

Should `background-color: transparent` be the component default?

I know it came up recently didn't it in this one:

* https://github.com/alphagov/govuk-frontend/issues/1625

Reply:
@colinrotherham that's a good point